### DISCUSSION TITLE
How many units is one dose of heparin in the inputevents table?

### DISCUSSION THREAD

**Discussion by gu-chun-feng** [MIT-LCP/mimic-code#1578] [MIMIC-IV]
In MIMIC-IV inputevents table.

such as:
itemid：225975，amount：1，amountuom：dose

How many units is one dose?
1dose = 100mg or 1dose = 12500u?

**Comment by heisenbug-1** [comment]
I'd also like to know this.  A significant number of patients have amount=1, amountuom=dose and itemid=225975 in my search, and I need to know, what the dosage is. I think itemid：225975 is under Heparin Prophylaxis label in d_items
